src/Cohesity.Powershell/Scripts/StorageDomain/Set-CohesityStorageDomain.ps1

```diff
@@ -49,19 +91,50 @@ function Set-CohesityStorageDomain {
 
         # Update the specified Storage domain
         if ($payload -ne $null) {
+            # Update the payload with specified parameter values
+            if ('UpdateField' -eq $PsCmdlet.ParameterSetName) {
+                $PsBoundParameters.keys | ForEach-Object {
+                    switch ($_) {
```

Comment:
Does this switch either of these fields.. the update can have all these fields right ? Does it handle that case ?

Reply:
Yes. It will update the all fields that user provided, it is irrespective of the count of parameters.